geoengine/workflow.py

```diff
@@ -500,6 +507,7 @@ def process_bytes(tile_bytes: Optional[bytes]) -> Optional[RasterTile2D]:
                 'spatialBounds': query_rectangle.bbox_str,
                 'timeInterval': query_rectangle.time_str,
                 'spatialResolution': str(query_rectangle.spatial_resolution),
+                'attributes': ','.join(map(str, bands))
```

Comment:
Why is a band called attribute?

Reply:
it's called `attributes` in the query rectangle for reasons of uniformity. The idea is that for raster queries you select bands and for vector queries you select columns and we needed a common name, because of the way the query struct is modeled in the backend.

Reply:
Should we call it multi-attribute and not multi-band then?

Reply:
call what multi-attribute?
right now, it's only called `attribute` in the `QueryRectangle` and `band` everywhere else. I'd rather stay with `band` for raster wherever it's possible, but we can discuss it with Johannes.

Reply:
Yeah, we should think of this. Maybe, for the user, it is confusion when we talk about attributes here and bands there.